web/styles/dark_theme.css

```diff
@@ -47,10 +47,7 @@
     #keyboard-shortcuts {
         & kbd {
             border: 1px solid var(--color-hotkey-hint);
-            border-radius: 3px;
```

Comment:
This border-radius should've been removed in the commit where it's specified in light theme.